### DISCUSSION TITLE
Cluster version mix

### DISCUSSION THREAD

**Discussion by hank-ht** [ravendb/ravendb#15881] [Q&A] [answered]
Is it possible to have a mix of versions in a cluster? Example we have a 3 node cluster running 4.2.x and we would like to upgrade to 5.x. Is it possible to do so one node at a time? If so, how long can we safely run nodes with a mix of 4.2 and 5.x? 

We won't be updating the client apps just yet as they will stay on 4.x.

**Comment by ayende** [comment, marked as answer]
Yes, that is a supported scenario. 
There is no time limit, for this, but obviously you'll not be able to use the new features until all nodes in the cluster match
